src/include/migraphx/permutation.hpp

```diff
@@ -0,0 +1,57 @@
+/*=============================================================================
```

Comment:
is this from another source? If so, do we need to update the copyright date?

Reply:
Oops this was auto-generated from my editor. I'll remove it.